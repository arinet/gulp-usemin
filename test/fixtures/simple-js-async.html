<!-- build:js app.js async -->
	<script src="js/lib.js"></script>
	<script src="js/main.js"></script>
<!-- endbuild -->