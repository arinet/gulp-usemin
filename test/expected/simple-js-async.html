<script src="app.js" async></script>